src/components/ThreadEnvelope.vue

```diff
@@ -632,16 +633,33 @@ export default {
 		filterSubject(value) {
 			return value.replace(/((?:[\t ]*(?:R|RE|F|FW|FWD):[\t ]*)*)/i, '')
 		},
+		onMessageLoaded() {
+			if (this.loadingBodyTimeout) {
+				clearTimeout(this.loadingBodyTimeout)
+				this.loadingBodyTimeout = undefined
+			}
+
+			this.loading = Loading.Done
+		},
 		async fetchMessage() {
-			this.loading = LOADING_MESSAGE
-			this.error = undefined
+			let loadingTimeout
+			const isCached = !!this.mainStore.getMessage(this.envelope.databaseId)
+			if (!isCached) {
+				loadingTimeout = setTimeout(() => this.loading = Loading.Skeleton, 200)
```

Comment:
eslint complains about the assignment as only expression of the array function

Reply:
Done